dataland-frontend/src/components/forms/parts/kpiSelection/KPIfieldSet.vue

```diff
@@ -0,0 +1,166 @@
+<template>
```

Comment:
Comment has nothing to do with this file:

I get this warning in the console when I visit the `https://local-dev.dataland.com/companies/xyz/frameworks/eutaxonomy-financials/uploadpage`   :


![image](https://user-images.githubusercontent.com/99255296/233019965-e760325e-45d6-4e11-854d-1ff52ada8fe3.png)

Reply:
changed
type: Number,